server/index.js

```diff
@@ -12,20 +13,21 @@ app.use(express.json());
 app.use('/:id', express.static(__dirname + '/../public'));
 
 //get review categories
-app.use('/api/overall_reviews', overallReviews);
+// app.use('/api/overall_reviews', overallReviews);
 
 //get individual reviews
-app.get('/api/individual_reviews/:id', (req, res) => {
-  Review.find({}, { user: 1, imageURL: 1, date: 1, reviewTxt: 1, _id: 0}, (err, results) => {
-    if (err) {
-      res.status(500).send(err);
-    } else {
-      res.send(results);
-    }
-  }).where('locationID').equals(req.params.id)
-    .sort({ date: -1 });
-});
+// app.get('/api/individual_reviews/:id', (req, res) => {
```

Comment:
do you really need this commented out part? maybe save it on a gist or retrieve it from the commit history if you still need it in the future?